### PR TITLE
docs: add guess the number idea and project

### PR DESCRIPTION
**Pull request by techtuner**
Hello,
Added idea for Machine Learning model for Image Classification

### CODE REVIEW

**Review by techtuner — COMMENTED** [review]
Added Guess the number game

File: data/ideas.json [hunk 1]
```diff
@@ -214,6 +214,12 @@
         "description": "App for share knowledge by donationg books.",
         "level": "intermediate",
         "tags": "book-donation"
+    },
+    {
+        "title": "Guess the Number",
+        "description": "Guess the number between 1 - 20",
+        "level": "basic",
+        "tags": "number-guessing, JavaScript, html, css"
```
Comment (ming-tsai) [inline data/ideas.json:222]: Please remove the programming language due to the idea could make in another language.

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @techtuner,
A suggestion for your pull request

File: data/ideas.json (hunk 1 above)
Comment (techtuner) [reply data/ideas.json]: I have removed the languages from ideas

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**